maven-failsafe-plugin/pom.xml

```diff
@@ -266,12 +266,12 @@
                                     <postBuildHookScript>verify</postBuildHookScript>
                                     <settingsFile>src/it/settings.xml</settingsFile>
                                     <skipInvocation>${skipTests}</skipInvocation>
-                                    <streamLogs>true</streamLogs>
                                     <showErrors>true</showErrors>
                                     <properties>
                                         <integration-test-port>${failsafe-integration-test-port}</integration-test-port>
                                         <integration-test-stop-port>${failsafe-integration-test-stop-port}</integration-test-stop-port>
                                     </properties>
+                                    <debug>true</debug>
```

Comment:
is this needed ?

Reply:
Sometimes I need to have debug logs if I investigate a problem. For instance failures of downloading artifacts of plugin parameters, etc.